lib/platform/github/index.spec.ts

```diff
@@ -2645,7 +2645,7 @@ describe('platform/github/index', () => {
       expect(res).toBeNull();
     });
     it('force-pushes with REST', async () => {
-      git.pushCommitAsRef.mockResolvedValueOnce();
+      git.createFunnyRenovateRef.mockResolvedValueOnce();
```

Comment:
Let's call them "funny" (as git and GitHub does) so we can instantly see what exactly refs we're meaning here or everywhere else